students/ABartles/Exercise6/test_mailroom.py

```diff
@@ -0,0 +1,67 @@
+from mailroom import *
+import os
+
+'''
+Items to Test:
+    sum_gifts
+    count_gifts
+    sorted donors
+    list_donors
+    *thanks letter
+    *send_thanks_all
+
+Items with user interaction: (can pytest test/simulate user imput?)
+    rec_donation
+    send_thanks_one
+    menu
+
+Not sure how to test w/ pytest:
+    create_report
+    run_func
+    __name__ == __main__
+'''
+
+def test_1():
+    assert sum_gifts("Anita Bath") == 350
+
+
+def test_2():
+    assert count_gifts("Seymour Butz") is 2
+
+
+def test_3():
+    name_list = ['Seymour Butz',
+                 'Anita Bath',
+                 'Isabelle Ringing',
+                 'John Smith']
+    name_list_sorted = sorted_donors()
+    for n in name_list:
+        assert n in name_list_sorted
```

Comment:
it should be more like `assert expected == name_list_sorted`

you're not actually testing your sorting logic.